README.md

```diff
@@ -46,16 +46,16 @@ export default Ember.Component.extend({
 You replace it with this:
 
 ```javascript
-import Ember from 'ember'
+import Component from '@ember/component'
 import { action, computed } from 'ember-decorators/object';
 import { service } from 'ember-decorators/service';
 
-export default class MyComponent extends Ember.Component {
```

Comment:
We should consider adding some blueprints that generate the new style and would add the classname. Definitely agree that we should use realistic class names